src/lib/viewers/__tests__/BaseViewer-test.js

```diff
@@ -278,6 +278,36 @@ describe('lib/viewers/BaseViewer', () => {
             expect(document.defaultView.addEventListener).to.be.calledWith('resize', base.debouncedResizeHandler);
             expect(base.addListener).to.be.calledWith('load', sinon.match.func);
         });
+
+        it('should load the annotator when load is emitted with scale', (done) => {
+            sandbox.stub(fullscreen, 'addListener');
+            sandbox.stub(document.defaultView, 'addEventListener');
+            sandbox.stub(base, 'loadAnnotator');
+            base.annotationsPromise = {
+                then: () => {
+                    done();
+                }
+            };
+
+            base.addCommonListeners();
```

Comment:
Not sure this is actually checking any expectations. I'd just do something like
```
this.annotationPromise = {};
base.emit('load', ....);
expect(base.loadAnnotor).to.be.called;
```
etc. for w/e is called and check that scale is set if it exists in event.scale